app/controllers/match_users_controller.rb

```diff
@@ -0,0 +1,32 @@
+class MatchUsersController < ApplicationController
+  def create
+    #you have a swipe
+    #swipe has a swiper and swipee and a restaurant
+    #pass swipe in here
+    @swipe = param_1
+    @match_user = MatchUser.create!(target_id: @swipe.swipee, creator_id: @swipe.swiper, status: 'Y', accepted: true)
+    redirect_to match_user_path(@match_user.id)
+
+    # if @new_match = MatchUser.create!(match_user_params)
+    #   redirect_to success_match_user_path(@new_match)
+    # else
+    #   redirect_to match_user_path
+    # end
+  end
+
+  def show
+    @match_user = MatchUser.find_by(id: params[:id])
+    @conversation = Conversation.create(sender_id: @match_user.creator_id, recipient_id: @match_user.target_id)
```

Comment:
Why do you create a conversation here? Won't you get a new one each time anyone looks at this MatchUser? Should it be on create?